src/update-feeds-loops/submit-transactions.ts

```diff
@@ -125,11 +125,11 @@ export const submitTransaction = async (
             logger.info(`Failed to submit replacement transaction because it was underpriced.`);
             return null;
           }
-          logger.info(`Failed to update a data feed.`, goMulticall.error);
+          logger.info(`Failed to submit the multicall transaction.`, goMulticall.error);
```

Comment:
The tx going through only means the RPC accepted our transaction, so the message is confusing.